tests/test_strds_sample.py

```diff
@@ -16,6 +16,12 @@
 __maintainer__ = "Soeren Gebbert"
 __email__      = "soerengebbert@googlemail.com"
 
+# TODO use modis data
```

Comment:
(same here)

Reply:
I think I have not change all, because it was not working